### PR TITLE
Blacked!

### PR DESCRIPTION
**Pull request by pedrocamargo**
This PR is just intended to start developing some code parameters standards for this repository as well

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.7
```
Comment (ychtw) [inline .pre-commit-config.yaml:6]: QGIS on mac still require python 3.6 now. Will this matter?

File: common_tools/__init__.py [hunk 2]
```diff
@@ -30,4 +30,4 @@
 from .report_dialog import ReportDialog
 from .worker_thread import WorkerThread
 from .about_dialog import AboutDialog
-from .all_layers_from_toc import all_layers_from_toc
\ No newline at end of file
+from .all_layers_from_toc import all_layers_from_toc
```
Comment (ychtw) [inline common_tools/__init__.py]: Looks the same as before?

**Review by ychtw — COMMENTED** [review]
One question about the python version.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (pedrocamargo) [reply .pre-commit-config.yaml:6]: No. This is only for running Black itself.

The Python 3.6/3.7 issue is something we will have to navigate, though, as for Windows it is already using Python 3.7.  We might have to look into Continuous Integration at some point soon.

Comment (ychtw) [reply .pre-commit-config.yaml:6]: Got it. Thanks for the clarification!

**Review by ychtw — APPROVED** [review]

**Pull request merged**